src/operations/hull/hull.js

```diff
@@ -0,0 +1,83 @@
+const flatten = require('../../utils/flatten')
+const areAllShapesTheSameType = require('../../utils/areAllShapesTheSameType')
+
+const { vec2 } = require('../../math')
+
+const { geom2, geom3 } = require('../../geometry')
+
+const hull2DPoints = require('./hull2DPoints')
+
+/*
+ * Create a convex hull of the given geom2 geometries.
+ * @param {...geometries} geometries - list of geom2 geometries
+ * @returns {geom2} new geometry
+ */
+const hullGeom2 = (...geometries) => {
+  geometries = flatten(geometries)
+
+  // extract the unique points from the geometries
+  let uniquepoints = []
+  geometries.forEach((geometry) => {
```

Comment:
since you are returning some data and not just iterating, a 'map' operation instead of forEach would be better I think ?

Reply:
Hmm... I guess. But then again it would not be readable due to the nesting of interations.